src/Framework/Telemetry/TelemetryBase.cs

```diff
@@ -13,12 +13,7 @@ internal abstract class TelemetryBase
     public abstract string EventName { get; }
 
     /// <summary>
-    /// Gets or sets a list of properties associated with the event.
+    /// Fetches current properties wrapped in Dictionary
     /// </summary>
-    public IDictionary<string, string> Properties { get; set; } = new Dictionary<string, string>();
-
-    /// <summary>
-    /// Translate all derived type members into properties which will be used to build <see cref="TelemetryEventArgs"/>.
```

Comment:
Please consider moving this comment to GetProperties

Reply:
Thank you! Updated the comment